controllers/admin/admin_mobile_controller.py

```diff
@@ -121,8 +122,8 @@ def post(self):
             return
 
         try:
-            clients = [int(c) for c in clients]
-            deferred.defer(NotificationHelper.send_broadcast, clients, title, message, url, app_version, _queue="admin")
+            client_types = [int(c) for c in client_types]
+            deferred.defer(TBANSHelper.broadcast, client_types, title, message, url, app_version, _queue="admin")
```

Comment:
This is going to be a problem - we can't defer static calls like this https://cloud.google.com/appengine/docs/standard/python/refdocs/google.appengine.ext.deferred.deferred